pkg/controller/manager/clusternetwork/controller.go

```diff
@@ -109,6 +129,33 @@ func (h Handler) deleteLinkMonitor(name string) error {
 	return h.lmClient.Delete(name, &metav1.DeleteOptions{})
 }
 
+func (h Handler) setNadReadyLabel(cn *networkv1.ClusterNetwork) error {
+	isReady := utils.ValueFalse
+	if cn.DeletionTimestamp != nil && networkv1.Ready.IsTrue(cn.Status) {
```

Comment:
what's the logic for cn.DeletionTimestamp != nil ?

Reply:
Set all net-attach-defs under the cluster network to be deleted as unready